Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/audit/AuditLogger.java

```diff
@@ -62,11 +64,13 @@
      */
     public void auditRequestMessage(T request, AssertionType assertion, NhinTargetSystemType target, String direction,
         String _interface, Boolean isRequesting, Properties webContextProperties, String serviceName) {
-
         LOG.trace("--- Before auditing of request message ---");
-        LogEventRequestType auditLogMsg = getAuditTransforms().transformRequestToAuditMsg(request, assertion, target,
-            direction, _interface, isRequesting, webContextProperties, serviceName);
-        auditLogMessages(auditLogMsg, assertion);
+        if (getAuditLogger() != null) {
+            getAuditLogger().auditRequestMessage(request, assertion, target, direction, _interface, isRequesting,
+                webContextProperties, serviceName, getAuditTransforms());
+        } else {
```

Comment:
You already log an error if JNDI lookup fails, as part of getAuditLogger() -- do we need this else statement?